Add speaker guidelines for meetups to Druid (#505)
apache/druid-website-src: src/pages/community/meetups.md

---
title: Druid Community Meetups
layout: simple_page
canonical: 'https://druid.apache.org/community/meetups'
---

## Druid Community Meetups

> Check out [Apache Druid on meetup.com](https://www.meetup.com/topics/apache-druid/) for links to regular

Reviewer: Drop "regular" I think.

meetups in cities all over the world.

> All Apache Druid Meetups are covered the Apache [anti-harassment policy](https://www.apache.org/foundation/marks/events#anti-harassment) and speakers are expected to follow the guidelines below.

Reviewer: "covered by"

Though @techdocsmith is there a cleaner grammar we could use?

### Speaker Guidelines

> **Skip the Sales Pitch:** Keep your talk unbiased and educational. It’s okay to mention commercial products you’re using, but only if they are a key part of your data architecture and Apache Druid. Content which is solely focused on promotion of a product or service is never permitted at an Apache Druid Meetup.

Reviewer: I think we need to be explicit that it's ok to mention things being possible in the free versions of products, and to emphasise that education - rather than persuasion - is the priority.

Reviewer: Oooh and also vendor neutrality - not sure how we fit this in tho?

Author: @petermarshallio what aspects of vendor neutrality are important to cover here?

> **Share Challenges:**  Feel good about sharing roadblocks you hit and how you worked around them – or ones you might still be struggling with – there may be a community member in the audience who can help you out! Engineering always involves trade-offs, so being realistic about them helps establish you as a credible advisor.

Reviewer: We could add some example topics? Like

- Making contributions (docs, code)
- Tutorials - things people can do on their own, whether it's for real or just experimentation, complete with re-usable code.
- Coaching - like tips and tricks, guidance on how to use features, and lessons learned.

> **Focus on the Technical:** This audience understands and has built data architectures – if you get it, they’ll get it. Of course, start with the basics of who you are, what you built, and why, and then go all-in on the technical insights, examples, and demos.

Reviewer: I like the Confluent "everyone with a mic should be technical" angle - should we strengthen that kind of view here?

**Show and Tell:** If you can, prepare a short demo to accompany your presentation to showcase your work. This will help put everything in context for your fellow developers and the more visual learners in the audience.

**Watch the Clock:** If you run over, then the next will, and so on, until we’re either completely off schedule or someone else’s talk gets cut short!  Practice your presentation ahead of time to make sure you stay within your allotted time.

> Need help with your a talk on Druid or have questions about Druid Meetups? Please reach out at [community@imply.io](mailto:community@imply.io).

Reviewer: Pull this and replace with a new channel in Druid Slack - what do you think @reenadotio @techdocsmith @abhishekagarwal87 ?

Reviewer 2: I think you can just add the Apache Druid slack channel. Or a link to where we tell people to sign up. We don't want to manage the invite code in multiple places IMO.

---
title: Druid Community Meetups
layout: simple_page
canonical: 'https://druid.apache.org/community/meetups'
---

## Druid Community Meetups

Check out [Apache Druid on meetup.com](https://www.meetup.com/topics/apache-druid/) for links to meetups in cities all over the world.

All Apache Druid Meetups subject to the Apache [anti-harassment policy](https://www.apache.org/foundation/marks/events#anti-harassment) and speakers are expected to follow the guidelines below.

### Speaker Guidelines

> **Skip the Sales Pitch:** Keep your talk unbiased and educational. It’s okay to mention commercial products you’re using, but only if they are a key part of your data architecture and Apache Druid or if what you are demonstrating can be accomplished with a free or free tier version of the commercial product or service. Content which is solely focused on promotion of a product or service is never permitted at an Apache Druid Meetup.

Reviewer 2: ```suggestion
**Skip the Sales Pitch:** Keep your talk unbiased and educational. It’s okay to mention commercial products you’re using only when they are a key part of your data architecture with Apache Druid, or your demonstration can work with a free or free tier version of the commercial product or service. Content which is solely focused on promotion of a product or service is never permitted at an Apache Druid Meetup.
```
Run on sentence

> **Share Challenges:**  Feel good about sharing roadblocks you hit and how you worked around them – or ones you might still be struggling with – there may be a community member in the audience who can help you out! Engineering always involves trade-offs, so being realistic about them helps establish you as a credible advisor.

Reviewer 2: ```suggestion
**Share Challenges:**  Feel good about sharing roadblocks you hit and how you worked around them – or ones you might still be struggling with. There may be a community member in the audience who can help you out! Engineering always involves trade-offs, so acknowledging your challenges helps establish you as a credible advisor.
```
Run on sentence.

> **Focus on the Technical:** This audience understands and has built data architectures – if you get it, they’ll get it. Of course, start with the basics of who you are, what you built, and why before going all-in on the technical insights, examples, and demos.

Reviewer 2: ```suggestion
**Focus on the Technical:** This audience understands and has built data architectures. If you get it, they’ll get it. Of course, start with the basics of who you are, what you built, and why. Then go all-in on the technical insights, examples, and demos.
```

> **Keep an open mind:** The Apache Druid project has [a vendor neutrality policy](https://community.apache.org/projectIndependence#:~:text=Apache%20projects%20are%20managed%20independently) because it is important to keep an open mind about approaches and not only share one product or solution when many are available. A community that obviously favors one specific vendor in some exclusive way will often discourage new contributors from competing vendors, and this would be an issue for the long-term health of the project.

Reviewer 2: ```suggestion
**Keep an open mind:** The Apache Druid project has [a vendor neutrality policy](https://community.apache.org/projectIndependence#:~:text=Apache%20projects%20are%20managed%20independently) because it is important to keep an open mind about different ways to solve problems. A community that obviously favors one specific vendor in some exclusive may discourage new contributors from competing vendors, potentially impacting the long-term health of the project.
```

> **Show and Tell:** If you can, prepare a short demo to accompany your presentation to showcase your work. This will help put everything in context for your fellow developers and the more visual learners in the audience.

Reviewer 2: ```suggestion
**Show and Tell:** If you can, prepare a short demo to accompany your presentation to showcase your work. This add context for your fellow developers and the more visual learners in the audience.
```

> **Watch the Clock:** If you run over, then the next will, and so on, until we’re either completely off schedule or someone else’s talk gets cut short!  Practice your presentation ahead of time to make sure you stay within your allotted time.

Reviewer 2: ```suggestion
**Watch the Clock:** If you run over, then the next will, and so on. Eventually the Meetup is completely off schedule , meaning someone's talk may be cut short!  Practice your presentation ahead of time to make sure you stay within your allotted time.
```
